src/pool/srv_iv.c

```diff
@@ -253,15 +253,46 @@ pool_iv_value_alloc(struct ds_iv_entry *entry, d_sg_list_t *sgl)
 	return pool_iv_value_alloc_internal(entry->iv_class->iv_class_id, sgl);
 }
 
+static int
+pool_iv_pre_sync(struct ds_iv_entry *entry, struct ds_iv_key *key,
+		 d_sg_list_t *value)
```

Comment:
Just curious, what is the difference between pool_iv_pre_sync and pool_iv_ent_update(). when will pool_iv_pre_sync be called?

Reply:
For pool map IV, pool_iv_pre_sync may make pool_iv_ent_update unnecessary. (I'm not 100% sure.) For other IVs, pool_iv_pre_sync is no-op.